src/Versatile_RotaryEncoder.cpp

```diff
@@ -19,9 +19,9 @@ Versatile_RotaryEncoder::Versatile_RotaryEncoder(uint8_t clk, uint8_t dt, uint8_
     pin_sw = sw;
 
     // Set encoder pins as inputs
-    pinMode(pin_clk, INPUT);
-    pinMode(pin_dt, INPUT);
-    pinMode(pin_sw, INPUT_PULLUP);
+    pinMode(pin_clk, INPUT_TYPE);
+    pinMode(pin_dt, INPUT_TYPE);
```

Comment:
Most encoders come already with pull up resistors, so I will test it first.

Reply:
Hello! you're right but there are also some bare encoder without board (I have one) and they need external pull up.
As the majority of encoder have their pullup resistors I set up default behaviour with external pull up